spring-cloud-generator/scripts/generate-steps.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+# Cleans the spring-cloud-previews folder and replaces its contents with an
+# empty setup (i.e. empty readme, no libraries, pom without modules)
+function reset_previews_folder() {
+  cd ${SPRING_GENERATOR_DIR}
```

Comment:
nit: we could avoid assuming `SPRING_GENERATOR_DIR` is set by relying on a function in this same file. Maybe it can fail if it's not set instead of defaulting to `pwd`, or maybe add a main comment section at the beginning of the file explaining that it needs to be set